c2bin/c2bin.c

```diff
@@ -28,8 +28,9 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <ctype.h>
 
-#define BUFFER_SIZE 80
+#define BUFFER_SIZE 1024
```

Comment:
Is this artificially limiting the input size? What happens if we pass a 2kb input to bin2c -h, then try to convert back with c2bin?

Reply:
This is limiting the length of the lines, you won't have a problem with a 2kb file passed through bin2c -h because the formating in bin2c places only 16 elements per line, but if the input comes with a different formating with longer lines we will have a problem.

Reply:
I'll takle this later when I get a chance.